Gem/Code/Source/Performance/HighInstanceExampleComponent.cpp

```diff
@@ -232,10 +212,23 @@ namespace AtomSampleViewer
         // pause script and tick until assets are ready
         ScriptRunnerRequestBus::Broadcast(&ScriptRunnerRequests::PauseScriptWithTimeout, 120.0f);
         AZ::TickBus::Handler::BusDisconnect();
+
+        if (m_testParameters.m_numShadowCastingSpotLights > 0 && m_diskLightsEnabled)
+        {
+            CreateSpotLights();
+        }
+
+        if (m_testParameters.m_activateDirectionalLight && m_directionalLightEnabled)
+        {
+            CreateDirectionalLight();
+        }
     }
 
     void HighInstanceTestComponent::OnAllAssetsReadyActivate()
     {
+        m_directionalLightFeatureProcessor = m_scene->GetFeatureProcessor<Render::DirectionalLightFeatureProcessorInterface>();
+        m_diskLightFeatureProcessor = m_scene->GetFeatureProcessor<Render::DiskLightFeatureProcessorInterface>();
```

Comment:
Activate() already does this, is it redundant?

Reply:
Good call, I will try to remove.